www/js/overlay.js

```diff
@@ -168,18 +168,24 @@ const resizeButtonFactory = (increaseFunc, decreaseFunc) =>
   );
 
 const copyURLButton = h(
-  'div.input-wrap',
+  'span.input-wrap',
   {
     title: `${url}`,
     onclick: () => {
       copy(url);
       analytics.trackEvent('overlay', 'urlCopied', url);
     },
   },
-  h('div.export-btn', h('i.fa.fa-files-o.cp-icon')),
-  h('div.setting-label', 'Copy URL'),
+  h('span.export-btn', { title: 'Copy URL' }, h('i.fa.fa-files-o.cp-icon')),
 );
 
+const overlayUrl = () =>
+  h(
+    'div.url-container',
+    h('input', { type: 'text', readOnly: true, value: getUrl() }),
```

Comment:
one thing I noticed is when the overlay is initially off, the url gets populated with the port being `undefined`.  
![image](https://user-images.githubusercontent.com/2982784/75456296-2f4afd00-592f-11ea-998b-c2b55ef3310c.png)

can we update the url on toggle of `Overlay On` ?